SPECS/kernel/config_aarch64

```diff
@@ -24,7 +24,7 @@ CONFIG_THREAD_INFO_IN_TASK=y
 # General setup
 #
 CONFIG_INIT_ENV_ARG_LIMIT=32
-# CONFIG_COMPILE_TEST is not set
+CONFIG_COMPILE_TEST=y
```

Comment:
Hi @hargar19 - This would only be required if we didn't want to enable `CONFIG_ARCH_TEGRA` for some reason. However, we already enable `CONFIG_ARCH_TEGRA`, so we should be able to leave `CONFIG_COMPILE_TEST` off as that option is mostly for build testing a wide range of drivers.

Reply:
Updated the PR.